src/libraries/System.IO.Hashing/src/System/IO/Hashing/Crc32.cs

```diff
@@ -169,6 +172,21 @@ public static int Hash(ReadOnlySpan<byte> source, Span<byte> destination)
 
         private static uint Update(uint crc, ReadOnlySpan<byte> source)
         {
+#if NET7_0_OR_GREATER
+            if (System.Runtime.Intrinsics.Vector128.IsHardwareAccelerated
+                && System.Runtime.Intrinsics.X86.Sse2.IsSupported
+                && System.Runtime.Intrinsics.X86.Pclmulqdq.IsSupported)
+            {
+                return UpdateX86(crc, source);
+            }
+#endif
+
+            return UpdateSlowPath(crc, source);
+        }
+
+        [MethodImpl(MethodImplOptions.AggressiveInlining)]
+        private static uint UpdateSlowPath(uint crc, ReadOnlySpan<byte> source)
```

Comment:
Let's call this UpdateScalar rather than UpdateSlowPath.

Reply:
Done